topics/reactiveui/index.md

```diff
@@ -0,0 +1,12 @@
+---
+created_by: Paul Betts
+display_name: ReactiveUI
+github_url: https://github.com/reactiveui/reactiveui
+logo: reactiveui.png
+released: 2009
+short_description: ReactiveUI is a composable, cross-platform model-view-viewmodel framework.
+topic: reactiveui
+url: https://reactiveui.net
+wikipedia_url: https://en.wikipedia.org/wiki/reactiveui
```

Comment:
This page doesn't seem to exist:

![reactiveui_-_wikipedia](https://user-images.githubusercontent.com/82317/31505104-b29714a2-af39-11e7-9371-15f02fd6efca.png)

The `wikipedia_url` field isn't required, you can just omit this line.

Reply:
I know, but working on that: https://github.com/reactiveui/ReactiveUI/issues/1521
If prefered, I can remove the link and add later.